sklearn/feature_extraction/hashing.py

```diff
@@ -7,9 +7,15 @@
 import numpy as np
 import scipy.sparse as sp
 
-from . import _hashing
+from ..utils import IS_PYPY
 from ..base import BaseEstimator, TransformerMixin
 
+if not IS_PYPY:
+    from ._hashing import transform as _hashing_transform
+else:
+    def _hashing_transform(*args, **kwargs):
+        raise NotImplementedError('FeatureHasher is not compatible with PyPy.')
```

Comment:
I think it would help if this error message could also include the URL of a github issue that explains why this cannot be currently be implemented with the current versions of PyPy & Cython.